source/_integrations/bmw_connected_drive.markdown

```diff
@@ -71,6 +73,44 @@ bmw_connected_drive:
       default: false
 {% endconfiguration %}
 
+## Notifications
+
+The `bmw_connected_drive` integration offers a notification service. Using this service you can send messages or Points of Interest (POI) to your vehicle. In your vehicle you can select this POI and the navigation will automatically start using the POI as a destination.
+The name of the service is `notify.bmw_connected_drive_<your_vehicle>`.
+
+### Examples
+
+A few examples on how to use the notification service.
+
+#### Send a text message to your vehicle
+
+```yaml
+...
+action:
+  service: notify.bmw_connected_drive_<your_vehicle>
+  data:
+    title: Message from Home Assistant # optional, will default to "Home Assistant" when left empty
+    message: The text of the message you want to send to your vehicle
+```
+
+#### Send a Point of Interest to your vehicle
+
+```yaml
+...
+action:
+  service: notify.NOTIFIER_NAME
```

Comment:
Here the service `notify.NOTIFIER_NAME` is called but above for sending messages, the service is called `notify.bmw_connected_drive_<your_vehicle>`.
I'd suggest to name both the same as both location and message are send via the same notify service.

Reply:
You're right, missed that one. Thanks 👍